src/decompiler-ast.h

```diff
@@ -44,22 +45,34 @@ struct Node {
   std::vector<Node> children;
   // Node specific annotations.
   union {
-    LabelType lt;  // br/br_if target.
-    const Var* var;  // Decl/DeclInit.
     struct { Index var_start, var_count; };  // FlushedVar/FlushToVars
-  };
+    const Var* var;  // Decl/DeclInit.
+    LabelType lt;  // br/br_if target.
+  } u;
```

Comment:
Because its nice to be able to do things like copy the entire union. Without the name, I need to pick the "biggest element", and if that ever changes, it breaks my code.